tests/e2e/test_e2e.py

```diff
@@ -295,3 +295,24 @@ def test_openai_parity_with_response_format():
         assert r.choices[0].delta.content == expected_out, (
             f"Server didn't return expected output.\n" f"OpenAI client output: {r}"
         )
+
+
+@e2e_from_file("tests/e2e/default_single_streaming.py")
+def test_e2e_single_streaming():
+    resp = requests.post("http://127.0.0.1:8000/predict", json={"input": 4.0}, headers=None, stream=True)
+    assert resp.status_code == 200, f"Expected response to be 200 but got {resp.status_code}"
+
+    outputs = []
+    for line in resp.iter_lines():
+        if line:
+            outputs.append(json.loads(line.decode("utf-8")))
+
+    assert len(outputs) > 1, "Expected multiple streamed outputs"
```

Comment:
```suggestion
```

Reply:
This is redundant since you already assert length == 3

Reply:
I agree. Will remove!